buildbot/master/files/config/master.cfg

```diff
@@ -179,8 +181,9 @@ c['builders'] = [
     DynamicServoBuilder("mac-rel-css", MAC_SLAVES, envs.build_mac),
     DynamicServoBuilder("mac-rel-intermittent", MAC_SLAVES, envs.build_mac),
     DynamicServoBuilder("mac-rel-wpt", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("windows-dev", WINDOWS_SLAVES, envs.build_windows),
-    DynamicServoBuilder("windows-nightly", WINDOWS_SLAVES, envs.build_windows),
+    DynamicServoBuilder("windows-dev-gnu", WINDOWS_SLAVES, envs.build_windows_gnu),
+    DynamicServoBuilder("windows-dev-msvc", WINDOWS_SLAVES, envs.build_windows_msvc),
```

Comment:
Our factories have extra logic for `"windows-*"` builders, and I don't think it would all apply to MSVC. You'll likely need to either change that logic or use a raw util.BuilderConfig with a different factory (similar to the doc builder).